js/projection/pg-chain.js

```diff
@@ -0,0 +1,24 @@
+import _ from 'underscore';
```

Comment:
It's better to separated all your new projections from the old ones. As they are conceptually different.
You can create a 'js/projection-chain' folder for them.